openapi/components/schemas/CurrencyCode.yaml

```
@@ -1,6 +1,6 @@
 type: string
 description: |-
-  ISO 4217 currency code.
+  Currency code in ISO 4217 format. Examples: `USD`, `EUR`, `CNY`.
```

```suggestion
  Currency code in ISO 4217 format.
```
The docs generate examples from defined examples already. We should generally not include examples in description.